src/App.tsx

```diff
@@ -1,6 +1,8 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
it is overall a bad practice to disable linting rules. But if you really need to do it - disable it only for the line where you need it instead of commenting the whole file
```suggestion
```